### DISCUSSION TITLE
Uninstall/redo setup

### DISCUSSION THREAD

**Discussion by mtnsec** [Security-Onion-Solutions/securityonion#3454] [Unsupported Versions] [answered]
Just following up on my previous question (https://github.com/Security-Onion-Solutions/securityonion/discussions/2331) and wondering if it's now possible to re-run setup in order to reconfigure SO or if a full reinstall is required?

The reason I ask is because I've converted my install to the Features version of Elasticstack, which at the time did not support TLS (so it's currently not running with TLS. though I'd like it to). When the new version of SO comes out that defaults to the Features version is released - will it automatically reconfigure my Elasticstack to use TLS or will I need to start over?  If I do need to start over, do I need to start from scratch with a full reinstall of all of my nodes or can I re-run setup to reconfigure?

**Comment by TOoSmOotH** [comment, marked as answer]
The change will be transparent to you. If you have any external services connecting to ES that are not part of SO then you will need to update them to use SSL. Otherwise, you are good to go!

**Reply by mtnsec** [reply]
Great! Thanks!
